### PR TITLE
sftp-config

### PR DESCRIPTION
**Pull request by oae**

### CODE REVIEW

File: .vscode/sftp.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "name": "openapi",
```
Comment (yasinuslu) [inline .vscode/sftp.json:2]: I'm assuming sftp plugin gets credentials from vscode settings. Can you provide your settings to be an example ?

Comment (oae) [reply .vscode/sftp.json:2]: First we need to install https://marketplace.visualstudio.com/items?itemName=liximomo.remotefs extension. Then we should put following configuration to settings.json

```json
"remotefs.remote": {
    "openapi": {
      "scheme": "sftp",
      "host": "<host>",
      "username": "<user>",
      "rootPath": "<remote-path>",
      "remotePath": "<remote-path>",
      "port": 22
    }
  },
```

**Comment by yasinuslu** [comment]
LGTM

**Comment by yasinuslu** [comment]
:+1:

**Comment by halitogunc** [comment]
👍

**Comment by yasinuslu** [comment]
:+1:

**Comment by halitogunc** [comment]
👍

**Comment by yasinuslu** [comment]
LGTM

**Comment by halitogunc** [comment]
lgtm

**Pull request merged**